libraries/node-core-library/src/Import.ts

```diff
@@ -267,28 +267,31 @@ export class Import {
       return path.resolve(normalizedRootPath, modulePath);
     }
 
-    if (includeSystemModules === true && Import._builtInModules.has(modulePath)) {
-      return modulePath;
+    // Built-in modules do not have a scope, so if there is a slash, then we need to check
+    // against the first path segment
+    const slashIndex: number = modulePath.indexOf('/');
+    const moduleName: string = slashIndex === -1 ? modulePath : modulePath.substr(0, slashIndex);
```

Comment:
I get deprecation warnings on `substr`, at least in web code. Prefer `slice` or `substring`.